runtime/oti/j9.h

```diff
@@ -346,6 +346,7 @@ static const struct { \
 #define J9_VALUETYPE_FLATTENED_SIZE(clazz)((UDATA) 0) /* It is not possible for this macro to be used since we always check J9_IS_J9CLASS_FLATTENED before ever using it. */
 #define IS_REF_OR_VAL_SIGNATURE(firstChar) ('L' == (firstChar))
 #define IS_QTYPE(firstChar) FALSE
+#define IS_LTYPE(firstChar) FALSE
```

Comment:
This is already defined at line 351 outside of `#ifdef J9VM_OPT_VALHALLA_VALUE_TYPES`

Reply:
oops, I meant to get rid of both of those

Reply:
fixed in adabd61